static_analyzer/program_info/modules.py

```
@@ -0,0 +1,93 @@
+            continue
+        crossing.setdefault((source_module, target_module), Counter())[edge.channel] += edge.weighted_value
+        boundary[source_module].add(edge.source)
+        boundary[target_module].add(edge.destination)
+        outgoing[source_module][edge.source] += edge.weighted_value
+        incoming[target_module][edge.destination] += edge.weighted_value
+    facts = {symbol.qualified_name: symbol for symbol in information.symbols}
+    profiles = []
+    for module_id, members in sorted(partition.items()):
+        flow = analyze_flow(information.edges, members, limit)
+        possible = len(members) * max(1, len(members) - 1)
+        rank = lambda values: tuple(name for name, _ in sorted(values.items(), key=lambda x: (-x[1], x[0]))[:limit])
```

Ruff (0.16.0)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `rank` as a `def`

(E731)